### ISSUE TITLE
Support for peer to peer Video streaming

### ISSUE THREAD

**Issue by SourceCodeZone** [aiortc/aiortc#356]
I need to transfer video from raspberry pi to remote web page which is  hosted in Ubuntu server. 
The raspberry Pi is in local network and no port forward   can done there, the Pi need to initiate the peer connection and send webcam data to the server web page which will access by the client using web-browser.
Is this possible with this library. Right now I am able install the **aiortc**  and able to run the sample.  

Note:
====
The server side code will be in javascript, so I can display the video on webpage like normal webrtc.
So basically I need to stream from PeerA(raspbarry-aiortc) to PeerB(webserver-html/javascript). 
And the webserver webRTC is implemented using **peerJs** https://peerjs.com/
And the signaling server will be **peerjs-server** https://github.com/peers/peerjs-server

**Comment by jlaine** [comment]
I don't understand your question: yes, WebRTC is peer-to-peer and this is what `aiortc` implements. For accuracy: you don't send video "to a web page", you send it on a peer connection. Whether or not that connection is initiated via a web page really depends on your environment.

**Comment by SourceCodeZone** [comment]
Thanks for the feedback. Actually my environment is like, Rpi in local network and the camera of the RPi need access from outside LAN, So the way I found is webRTC, where Rpi(first peer) need to initiate the connection to the central server(second peer). So my doubt was is there every functionality available with aiortc like normal JavaScrip webRtc. https://www.tutorialspoint.com/webrtc/webrtc_rtcpeerconnection_apis.htm
